LiskHQ/lisk-hub
# Animate shape colors in registration - Closes #703

### What was the problem?
The shapes' colors were not animated

### How did I fix it?
By changing the gradients depending on the progress percentage + fading some of them out towards the end

### How to test it?
Try to register a new account, in the create step you should see the color gradient animation

used scheme:
<img width="764" alt="screen shot 2018-04-09 at 18 18 39" src="https://user-images.githubusercontent.com/9592216/38509683-eb510e08-3c22-11e8-95bc-5fddafe3fd84.png">

![screen shot 2018-04-09 at 18 21 07](https://user-images.githubusercontent.com/9592216/38509699-fb5c6446-3c22-11e8-8896-4888a2524c53.png)

### Review checklist
- The PR solves #703
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
